src/twisted/internet/test/test_inlinecb.py

```diff
@@ -126,12 +127,13 @@ def inline():
 
 
 class ForwardTraceBackTests(SynchronousTestCase):
-    def test_forwardTracebacks(self):
+    @skipIf(sys.version_info > (3, 12), "applies to Python 3.12 and older")
```

Comment:
Trying to have some "standard" skip reason.

```suggestion
    @skipIf(sys.version_info > (3, 12), "Needs Python 3.12 or older")
```